tests/smoke_tests/run_smoke_test.py

```diff
@@ -19,6 +21,11 @@ async def run_smoke_test(
     client_python_path: str,
     config_path: str,
     dataset_path: str,
+    checkpoint_path: Optional[str] = None,
+    assert_evaluation_logs: Optional[bool] = False,
+    # The param below exists to work around an issue with some clients
+    # not printing the "Current FL Round" log message reliably
```

Comment:
I assume this is due to the fact that some clients (for example the evaluation clients) don't print this log? Are there other clients that don't log this message? If so, maybe we can edit them to make sure they do.

Reply:
Not that only. I noticed the examples in the `dp_fed_examples` folder don't reliably print that message. Sometimes they print everything and some other times some of the messages are missing, which made the test fail intermittently, so I added that option to mitigate the issue. I haven't noticed that in the other examples. I don't know if there is anything in those examples that would contribute to that behaviour, I didn't go as far as investigating that in detail. Any clues?

Reply:
Interesting. I don't know the issue here, but we can investigate later. I plan to rework some of the privacy accounting soon anyway.